### ISSUE TITLE
Key property on Multiselect undefined when set (@next)

### ISSUE THREAD

**Issue by Falanyx** [monterail/vue-multiselect#92]
In the 2.0 (next) version of vue-multiselect, the key property on the multiselect element is undefined even when set explicitly. This is breaking the functionality of :selected when the value of that refers to an array of objects, allowing doubling up of selected elements.

Setting the value manually via the browser console with `$vm.key = "id"` causes the element to behave as expected.

Vue Devtools screenshot:
<img width="1671" alt="screen shot 2016-08-24 at 11 52 09 am" src="https://cloud.githubusercontent.com/assets/11275955/17943769/b0987440-69f1-11e6-9f42-3e6f88748ab5.png">

Usage code:
```html
       <multiselect
         :options="companies"
         :multiple="true"
         :selected="selected_companies"
         @update="updateCompanies"
         key="id"
         label="label"
       ></multiselect>
```

Versions:
    "vue": "^2.0.0-rc.3",                                                                                                 
    "vue-multiselect": "^2.0.0-beta.2",

**Comment by Falanyx** [comment]
Vue 2.0 added special (reserved) attributes `key`, `ref`, and `slot` (see vue/vuejs issue 2873). Chances are the addition of the `key` reserved attribute is the issue here. This property may require a name change in vue-multiselect as a result.

**Comment by shentao** [comment]
Yeah, it must it. Will fix it ~sunday. Thanks!

**Comment by shentao** [comment]
Should be fixed in 2.0.0-beta.3

**Comment by Falanyx** [comment]
Confirmed working in 2.0.0-beta.3 with `trackBy` prop. Thank you!
